### PR TITLE
Enable the python logging and set default level to ERROR.

### PR DESCRIPTION
**Pull request by weiqiand**
If want to DEBUG level, change the logging.basicConfig(level=logging.DEBUG)

### CODE REVIEW

**Review by satish153 — COMMENTED** [review]
Why are prints for errors not changed to logging.error?
For example: 
print("CLI Error: get table name failed. [{}]".format(self._cintf.err_str(sts)))

**Comment by weiqiand** [comment]
This enhancement only address the new debug print statement.
Yes. We can do that later with this python logging frameworks.
thanks,
Weiqian

**Review by satish153 — APPROVED** [review]

File: tdi_python/tdiTable.py [hunk 1]
```diff
@@ -1855,7 +1856,7 @@ def get_type(self):
         table_type = c_int(-1)
         sts = self._cintf.get_driver().tdi_table_type_get(self._info_handle, byref(table_type))
         #print("table name= "+str(self.name)+" table_type="+str(table_type))
-        print("table_type=0x%x" % table_type.value)
+        logging.debug("table_type=0x%x" % table_type.value)
```
Comment (saynb) [inline tdi_python/tdiTable.py:1859]: I suggest let's just remove these. I don't think they are providing any real value

Comment (weiqiand) [reply tdi_python/tdiTable.py]: done

**Pull request merged**